docs/orchestrating-elastic-stack-applications/logstash.asciidoc

```diff
@@ -0,0 +1,569 @@
+:page_id: logstash
+:logstash_recipes: https://raw.githubusercontent.com/elastic/cloud-on-k8s/{eck_release_branch}/config/recipes/logstash
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Run Logstash on ECK
+
+experimental[]
+
+This section describes how to configure and deploy Logstash with ECK.
+
+* <<{p}-logstash-quickstart,Quickstart>>
+* <<{p}-logstash-configuration,Configuration>>
+** <<{p}-logstash-configuring-logstash,Configuring Logstash>>
+** <<{p}-logstash-pipelines,Configuring Pipelines>>
+** <<{p}-logstash-pipelines-es,Using Elasticsearch in Logstash Pipelines>>
+** <<{p}-logstash-expose-services,Exposing Services>>
+* <<{p}-logstash-configuration-examples,Configuration examples>>
+* <<{p}-logstash-advanced-configuration,Advanced Configuration>>
+** <<{p}-logstash-jvm-options,Setting JVM Options>>
+* <<{p}-logstash-technical-preview-limitations,Technical Preview Limitations>>
+
+
+NOTE: Running Logstash on ECK is compatible only with Logstash 8.7+.
```

Comment:
Do we need to remove this? I do see LS 8.6.1 versions are in settings (ex: [logstash.yml](https://github.com/elastic/cloud-on-k8s/blob/feature/logstash/config/samples/logstash/logstash.yaml#L7)) and as I understood stack monitoring for Logstash requires 8.7.0+. 
Or, we can change all configs version to 8.7.0 to align with this note.